packages/app/src/runner/iframe-model.ts

```diff
@@ -247,6 +248,35 @@ export class IframeModel {
 
   _storeOriginalState () {
     const autStore = useAutStore()
+
+    if (!this.isAUTSameOrigin()) {
```

Comment:
original in [runner-shared](https://github.com/cypress-io/cypress/pull/18075/files#diff-1344127f3f743520c7bb12af5f96a5953484a13d551fc9902bee690c99f9ee81R222) in 9.x. Not related here, but `removeSrc` can be removed from the `develop` branch